ek-integration/expertkit_vllm/tests/test_vllm.py

```diff
@@ -1,21 +1,25 @@
 from vllm import LLM, SamplingParams
 import os
 
-os.environ["EXPERTKIT_ENABLE"] = "1"
 os.environ["VLLM_MLA_DISABLE"] = "1"
-os.environ["EXPERTKIT_DEBUG_MODE"] = "1"
-os.environ["EXPERTKIT_MODE"] = "moe_mode"
+
+os.environ["EK_ENABLE"] = "0"
+os.environ["EK_MODEL_NAME"] = "qwen3"
+os.environ["EK_MODE"] = "expert_mode"
+os.environ["EK_ADDR"] = "localhost:5002"
+os.environ["EK_CLIENT_TIMEOUT"] = "2"
+os.environ["EK_DEBUG_MODE"] = "0"
 
 prompts = [
     "Hello, my name is",
     "The president of the United",
     # "The capital of France is",
     # "The future of AI is",
-]
+] * 32
 sampling_params = SamplingParams(temperature=0.8, top_p=0.95)
 
 llm = LLM(
-        model="/mnt/xact/kioxia/.cache/huggingface/hub/DeepSeek-R1",
+        model="/home/liucp/Documents/gitRepos/expert-kit/expert-kit-deploy/data/qwen3/qwen3",
```

Comment:
```suggestion
        model=args.model_path,
```

use parameter or env variable here